test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/wwwroot/testRunner.js

```diff
@@ -126,6 +126,23 @@ export function assertLayerExists(methodName, layerType) {
     throw new Error(`Expected layer of type ${layerType} but found none`);
 }
 
+export function assertObjectHasPropertyWithValue(methodName, objectId, propertyName, expectedValue) {
+    let props = propertyName.split('.');
+    let obj = arcGisObjectRefs[objectId];
+    for (var i = 0; i < props.length; i++) {
+        let prop = props[i];
+        let candidate = obj[prop];
+        if (candidate === undefined) {
```

Comment:
Occasionally, I get errors with specific inputted property names and values. I will continue to test this method refine the test entries